src/main/kotlin/no/skatteetaten/aurora/boober/service/openshift/OpenShiftClient.kt

```diff
@@ -55,8 +55,8 @@ data class OpenshiftCommand(
     }
 
     fun setWebsealDone(): OpenshiftCommand {
-        val annotations = payload.get("metadata").get("annotations") as ObjectNode
-        annotations.replace(WEBSEAL_DONE_ANNOTATION, TextNode(previous?.annotation(WEBSEAL_DONE_ANNOTATION)))
+        val annotations = payload.get("metadata").get("annotations") as ObjectNode?
```

Comment:
kan vel egentlig returnere hvis den er null. dvs `return this` da slipper man copy i de tilfellene.

Reply:
God poeng, har oppdatert